projects/admin/src/app/record/detail-view/item-detail-view/item-transaction/item-transaction.component.ts

```diff
@@ -189,6 +189,11 @@ export class ItemTransactionComponent implements OnInit, OnDestroy {
       )
       .subscribe((itemData: any) => {
         this.removeRequest.emit(true);
+        this._itemService.updateItem$.next(itemData.pid);
+        this._toastrService.warning(
+          this._translateService.instant('The item is ' + itemData.status),
```

Comment:
Instead of `+` you can use variable. See https://github.com/rero/rero-ils-ui/blob/2e98904fbfeca8b4eb27862bc9238cc982334287/projects/admin/src/app/routes/import-documents-route.ts#L93 for an example.

Reply:
Using a variable {{status}} displays `{{status}}` in the message. Not the real status. Is this `stream` really applicable here?
```
return this._translateService.stream('{{ total }} results of {{ remoteTotal }}',
      {
        total,
        remoteTotal: hits.remote_total
      }
    );
```